desc/configuration.py

```diff
@@ -632,13 +610,14 @@ def change_resolution(self, L=None, M=None, N=None, *args, **kwargs):
         self.Z_basis.change_resolution(self.L, self.M, self.N)
         self.L_basis.change_resolution(self.L, self.M, self.N)
 
+        if L_change:
+            self._pressure.change_resolution(L=L)
```

Comment:
@ddudt these only work for PowerSeriesProfiles, is there a way to make it more general?

Reply:
Only update profile resolutions if applicable (ie. if a PowerSeries)

Reply:
can change to if hasattr(pressure, change...):

Reply:
and make this `self.pressure`